keyvi/src/cpp/dictionary/dictionary.h

```diff
@@ -70,7 +70,7 @@ final {
     }
 
     // temporary implementation
-    fsa::automata_t GetFsa() {
+    fsa::automata_t GetFsa() const {
```

Comment:
I suggest have this getter without `const` qualifier, otherwise its possible to get the pointer from const Dictionary and modify the internal state from outside.